python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -39,25 +38,36 @@ public class IPythonTreeMakerTest extends RuleTest {
   public void line_magic() {
     var parse = parseIPython("print(b)\n" +
       "a = %alias showPath pwd && ls -a\n", treeMaker::fileInput);
-    assertThat(parse).isNotNull();
-    var lineMagic = findFirstChildOf(parse, Tree.Kind.LINE_MAGIC);
+    var statementList = findFirstChildWithKind(parse, Tree.Kind.STATEMENT_LIST);
+    assertThat(statementList.children()).hasSize(2);
```

Comment:
(Minor) You can probably declare a variable instead of repeating these `statementList#children` calls. Feel free to ignore though. Just saying this as I saw the same call was often repeated.
Though it may be error prone, since that's another reassignment that's done between each test case in the same method (maybe it'd be better to split and have 1 snippet per method?).
I let you decide if you prefer to keep things the way they are.